### PR TITLE
B2B Add payment limitation info around purchase orders

### PR DESCRIPTION
**Pull request by dobooth**
## Purpose of this pull request

This pull request (PR) adds information about limitations around payments when processing purchase orders.

## Magento release version

- 2.4.2

## Product editions

- B2B extension

## Affected documentation pages

https://docs.magento.com/user-guide/customers/account-dashboard-my-purchase-orders.html

### CODE REVIEW

File: src/customers/account-dashboard-approval-rules.md [hunk 1]
```diff
@@ -18,6 +18,20 @@ To access the approval rules for your company, click **Approval Rules** in the l
...
+
+Purchase order approval supports both online and offline payment methods.
+All default offline payment methods are supported for purchase order approvals.
+
+For online payments, the following methods are supported:
+
+* Paypal Express
+* Braintree payments
+
+{:.bs-callout-warning}
```
Comment (leslietilling) [inline src/customers/account-dashboard-approval-rules.md:31]:
```suggestion
{:.bs-callout-info}
```

File: src/customers/account-dashboard-approval-rules.md [hunk 2]
```diff
@@ -18,6 +18,20 @@ To access the approval rules for your company, click **Approval Rules** in the l
...
+Purchase orders should use an "In-Context Checkout Experience". Out-of-Context checkouts are not supported as they bypass the normal checkout flow.
+Generally, "In-Context" means you stay on the Magento site to complete the process. "Out-of-Context" is when you are taken to another site to complete the purchase.
+Also, using gift cards, store credit, or reward points with online payment methods for purchase orders is not supported. Enabling these features with online payments may cause some unexpected behavior. It is recommended to disable gift cards, store credit, and reward points when online payments are enabled for purchase orders.
```
Comment (leslietilling) [inline src/customers/account-dashboard-approval-rules.md:34]:
```suggestion

{:.bs-callout-warning}
Using gift cards, store credit, or reward points with online payment methods for purchase orders is not supported. Enabling these features with online payments may cause some unexpected behavior. It is recommended to disable gift cards, store credit, and reward points when online payments are enabled for purchase orders.
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
I would make the context information a general note and leave the other as a warning. In this case I think we can put one after the other.

File: src/customers/account-dashboard-approval-rules.md [hunk 3]
```diff
@@ -18,6 +18,20 @@ To access the approval rules for your company, click **Approval Rules** in the l
 _Company Approval Rules_
 
 Depending on the company role and permissions, users may have the option to edit, delete, or view approval rules.
+## Purchase order payment methods
```
Comment (leslietilling) [inline src/customers/account-dashboard-approval-rules.md:21]: Need a blank before the heading

```suggestion

## Purchase order payment methods
```

**Comment by dobooth** [comment]
I suspected I was overloading that warning...

File: src/customers/account-dashboard-approval-rules.md [hunk 4]
```diff
@@ -19,6 +19,23 @@ _Company Approval Rules_
...
+All default offline payment methods are supported for purchase order approvals.
+
+For online payments, the following methods are supported:
+
+* Paypal Express
+* Braintree payments
+
+{:.bs-callout-info}
+Purchase orders should use an _In-Context_ checkout experience. _Out-of-Context_ checkouts are not supported because they bypass the normal checkout flow.
+Generally, _In-Context_ means the customer stays on the Magento site to complete the process. _Out-of-Context" is when the customer is taken to another site to complete the purchase.
```
Comment (leslietilling) [inline src/customers/account-dashboard-approval-rules.md:34]: My boo boo from last round

```suggestion
Generally, _In-Context_ means the customer stays on the Magento site to complete the process. _Out-of-Context_ is when the customer is taken to another site to complete the purchase.
```

**Review by leslietilling — APPROVED** [review]
Just that minor fix and good to go!  Let's make sure to squash :)

**Comment by dobooth** [comment]
running tests

**Comment by leslietilling** [comment]
@dobooth  Let's add a "Whats New" for this one.

**Pull request merged**